triage/summarize.py

```diff
@@ -484,6 +484,7 @@ def render(builds, clustered):
 
 SIG_LABEL_RE = re.compile(r'\[sig-([^]]*)\]')
 
+# pylint: disable=too-many-branches
```

Comment:
Doesn't this disable this lint for the whole file? If its just annotate_owners isn't it more appropriate to inline it there?

Reply:
Oops. My intention was to **disable** only for `annotate_owners` function. Let me update to an inline comment.